infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -191,6 +193,9 @@
     // Error code required to detect an ungraceful shutdown.
     private static final int EXIT_ERROR_CODE = 100;
 
+    private static final String DEFAULT_METRICS_LOGGER_NAME = "CorfuMetrics";
+
+    private static final Duration DEFAULT_METRICS_LOGGING_INTERVAL_DURATION = Duration.ofSeconds(1);
```

Comment:
This is a really short period given that most of the metrics collected are aggregates.

Reply:
This was for debugging. Increased to a minute.